### PR TITLE
Perfomance stats for the trainee

### PR DESCRIPTION
**Pull request by Christian-Ishimwe**
### What does this PR do?
This PR implement the perfomance statistics for the trainee
### Description of the task to be completed
At each sprint the ttl rate the trainee, through quality, quantity and proffessionalism. Once the trainee is rated, he will see the results on the dashboard homapage as well as on perfomance screen.
### How this should be tested?

- clone this repo
- after cloning, checkout to `git checkout ft-statistics`
- Install the packages by `npm install`
- set the `.env`  as described in `.env.example`
- run `npm run start` for running the server, for the preference, you might use the android studio simulator or expo mobile app
- By using the web version, sign in as a ttl and try to rate a trainee
-

### CODE REVIEW

**Comment by fredshema** [comment]
@Christian-Ishimwe please add a description to your pr

File: app/dashboard/perfomance/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Text, View } from 'react-native';
+import PerformanceScores from '@/components/perfomanceStats';
+export default function TraineeDashboard() {
+  return (
+    <View>
+      <Text>Trainee Dashboard Coming Soon</Text>
```
Comment (pacifiquemboni) [inline app/dashboard/perfomance/index.tsx]: You can remove this as long as we have something to display
<Text>Trainee Dashboard Coming Soon</Text>

**Review by niyibi250 — COMMENTED** [review]
if you give as  credential for rated tranees  it will be best for testing the feature

**Review by niyibi250 — APPROVED** [review]

**Review by MugemaneBertin2001 — APPROVED** [review]

**Review by pacifiquemboni — APPROVED** [review]

**Review by Mbabazi12 — APPROVED** [review]

**Review by ncutixavier — COMMENTED** [review]
I can't review this PR due to many unnecessary changes, Please includes only necessary changes, Align well you components, ...

`dashboard/homepage/PerformanceStats.tsx`

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**